CHANGELOG.md

```diff
@@ -13,6 +13,10 @@ Possible log types:
 
 ### Unreleased
 
+### v0.5.0 (2019-12-30)
```

Comment:
Date needs updating.

Reply:
Hehe, we only needed slightly more than 3 month to get the release done :laughing: